packages/components/PersonaCoin/src/PersonaCoin.types.ts

```diff
@@ -60,6 +61,17 @@ export type PersonaCoinFluentColor =
 
 export type PersonaPresence = 'none' | 'offline' | 'online' | 'away' | 'dnd' | 'blocked' | 'busy';
 
+export type RingThickness = number | 'xSmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge';
+
+export interface IRingConfig {
+  accent?: boolean;
+  transparent?: boolean;
+  ringColor?: Color; // glow
+  ringBackgroundColor?: Color; // inner/outer rings
+  ringThickness?: RingThickness;
+  innerGap?: RingThickness;
+}
```

Comment:
Can this just a type called RingProps?

Reply:
ring is a prop to the Avatar / PersonaCoin, and these values are used to create the styles for the ring slots. So calling it a RingConfig seems more appropriate =)

Reply:
Alright. 
A note: CXE is moving away from using "I" as a prefix for interfaces. so "RingConfig" is fine.